climada/engine/impact.py

```diff
@@ -1390,17 +1390,25 @@ def _selected_exposures_idx(self, coord_exp):
         return sel_exp
 
     @classmethod
-    def concat(cls, imp_list,concat_type='time_window'):
+    def concat(cls, imp_list, concat_type='time'):
         """Concatenate impact objects of the same hazard type, frequency,
-        exposure, and coordinate system.
+        exposure, and coordinate system. This function is useful if, e.g. different impact functions
+        have to be applied for different seasons (e.g. for agricultural impacts).
+
+        NOTES
+        -----
+            - Concatenation of impacts with different exposure (e.g. different countries)
+                could also be implemented here in the future.
+            - For now, impacts have to be computed based on the same hazard set
+                that is split into time windows using hazard.select()
 
         Parameters
         ----------
         imp_list : list of climada.engine.Impact objects
                 list of Impact object to concatenate
         concat_type: string
                 string specifying the type of concatenation to be done.
-                    'time_window': concatenate impacts for different events / time slices
+                    'time': concatenate impacts for different events / time slices
```

Comment:
I suggest to call it `event` then.

Reply:
resolved as the "exposure" concatenation option is not implemented anyways the user does not have specify a  `concat_type`  now